support/openharmony/oh-package.json5

```diff
@@ -0,0 +1,12 @@
+{
+  "license": "MPL",
```

Comment:
Can we keep the license of the ArkTS project under Apache-2?

Reply:
Done in [e0b51fb](https://github.com/servo/servo/pull/33295/commits/e0b51fb229acbabdb6d2f192cea530fa7d151538)